frontend-react/src/components/header/OrgDropdown.tsx

```diff
@@ -0,0 +1,42 @@
+import { Dropdown } from "@trussworks/react-uswds";
+import { CSSProperties, useState } from "react";
+import { useResource } from "rest-hooks";
+import OrganizationResource from "../../resources/OrganizationResource";
+import { GLOBAL_STORAGE_KEYS, useGlobalContext } from "../GlobalContextProvider";
+
+const OrganizationDropdown = () => {
+    const [org, setOrg] = useState(localStorage.getItem(GLOBAL_STORAGE_KEYS.GLOBAL_ORG) || "")
+    const { updateOrganization } = useGlobalContext();
+
+    let orgs = useResource(OrganizationResource.list(), {
+        sortBy: undefined,
+    }).sort((a, b) => a.name.localeCompare(b.name));
+
+    let setValue = (e: any) => {
+        setOrg(e)
+        updateOrganization(e)
+        window.location.reload()
```

Comment:
instead of reloading, thoughts on sending users to the home page? if we reload and their on a report page, they'll probably get and error/crash. open to other ideas!

Reply:
I wasn't getting any crashes locally, is there something about the staging/prod env that might prompt an error?

Reply:
OHHHH I get what you're saying; if it's on `/report-details/...` it'll error out. I think catching that and bumping them back to `/daily` would be a better experience than all the way home, but we'll have to build out that check for any detail-related pages on the sender side as well (i.e. bumping them back to `/upload` or wherever the base of their experience lies).

Reply:
This works! It might start to get gnarly if/when there are a lot of pages we need to make exceptions for. Can cross that bridge when we get to it though